ERDDAP_timeseries_explorer.ipynb

```diff
@@ -14,219 +14,10 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "import numpy as np\n",
```

Comment:
@apkrelling with this change the notebook is pretty much a call to the underlying app, which is what we wanted! However, some folks who are not GitHub savvy may have difficulty finding the code. Maybe we should add a link to the repo in the header of the notebook?

Reply:
done!